### PR TITLE
I modified readme.md, index.html,styles.css, I also added images.

### PR DESCRIPTION
**Pull request by Eugeneiyukhun**
I built a portfolio with mobile and desktop views using
- Linter
- HTML and CSS
- flex.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -44,5 +44,178 @@ <h2><img src="image/LET’S CONNECT.svg" alt=""></h2>
...
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+        </section>
+        <section class="section3">
+            <div class="about">
+                <h1>About Myself</h1>
```
Comment (leonardodiasb) [inline index.html]: - [ ] This is the title of this section, but not the title of the page. Usually, the page has one main title and no more than one `h1` tag. Please change this to a `h2` and read [this article](https://www.reliablesoft.net/h1-tag/) for a better understanding of the headings and SEO.

File: styles.css [hunk 2]
```diff
@@ -97,3 +97,378 @@ body {
...
+  border-radius: 8px;
+  display: flex;
+}
+
+.section3 {
+  background-color: white;
+  border-top-right-radius: 7rem;
+  margin-top: 20px;
+  padding: 20px 5%;
+}
```
Comment (leonardodiasb) [inline styles.css:343]: - [ ] This section is great and very similar to the Figma template, but it is a project requirement to style it with Flexbox. Please, add flexbox to this class to follow all the instructions.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Eugeneiyukhun, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Correct use of Github flow. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -44,5 +44,178 @@ <h2><img src="image/LET’S CONNECT.svg" alt=""></h2>
...
+            </div>
+            <div class="get-resume">
+                <button type="button" class="get-resume-btn">Get My Resume</button>
+            </div>
+
+            <div class="accordion">
+                <ul class="mother-accordion">
+                    <li class="accordion-item">
+                        <h2>
+                            <span>Languages</span>
```
Comment (aimalamiri) [inline index.html:180]: It looks amazing 👌but you can improve it by following the exact Figma design. Please make sure to fix the font-weight here. These kinds of simple things can improve your design significantly.

image

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @Eugeneiyukhun, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

AlmostThereReachGIF


 ## Highlights

- The mobile version is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @Eugeneiyukhun , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :heavy_check_mark: 
- [ ] Nice UI design :white_check_mark: 
- [ ] Implemented changes from previous reviewers :white_check_mark: 
- [ ] Linters are set up correctly and passing :white_check_mark: 
- [ ] Good descriptive commit messages :white_check_mark: 
- [ ] Descriptive pull request and summary :white_check_mark: 



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**